env0/resource_configuration_variable_test.go

```diff
@@ -0,0 +1,81 @@
+package env0
+
+import (
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestUnitConfigurationVariableResource(t *testing.T) {
+	resourceType := "env0_configuration_variable"
+	resourceName := "test"
+	resourceFullName := resourceAccessor(resourceType, resourceName)
```

Comment:
```suggestion
	accessor := resourceAccessor(resourceType, resourceName)
```
to fit new structure

Reply:
3d5100ff